src/helpers/groupDataHelpers.js

```diff
@@ -50,6 +52,31 @@ export function loadProjectGroupData(toolName, projectDir) {
   return project.getGroupsData(toolName);
 }
 
+/**
+ * gets path to a tool's group data from the project.
+ * @param {string} toolName - the name of the tool who's helps will be loaded
+ * @param {string} projectDir - the absolute path to the project
+ * @param {string} groupID - group identifier
+ * @returns {string} path to a tool's group data
+ */
+export function getGroupDataIndexPath(toolName, projectDir, groupID) {
+  const project = new ProjectAPI(projectDir);
+  return path.join(project.getCategoriesDir(toolName), groupID + '.json');
+}
+
+/**
+ * gets path to a tool's group data from the project.
+ * @param {string} toolName - the name of the tool who's helps will be loaded
+ * @param {string} projectDir - the absolute path to the project
+ * @param {string} groupID - group identifier
+ * @param {object} groupData - data to save
+ * @returns {string} path to a tool's group data
+ */
+export function saveGroupData(toolName, projectDir, groupID, groupData) {
+  const dataPath = getGroupDataIndexPath(toolName, projectDir, groupID);
+  return fs.outputJson(dataPath, groupData);
+}
```

Comment:
```suggestion
  return fs.outputJson(dataPath, groupData, { spaces: 2 });
}
```
Any reason we are not using outputJsonSync instead?

Reply:
Yes - we don't need to wait for it to complete.